cmd/istioctl/main.go

```diff
@@ -183,6 +213,10 @@ var (
 		Use:   "list <type>",
```

Comment:
why do we need to support list? Since we don't have a release, lets not worry about deprecation